c/str/split_into_nhot.cc

```diff
@@ -165,6 +166,21 @@ DataTable* split_into_nhot(Column* col, char sep) {
 
   oem.rethrow_exception_if_any();
 
+  // Re-order columns, so that column names go in alphabetical order.
+  if (sort) {
+    size_t ncols = outcols.size();
+    std::vector<std::string> outnames_sorted(ncols);
+    std::vector<Column*> outcols_sorted(ncols);
+    std::vector<size_t> colindex = sort_index<std::string>(outnames);
+    for (size_t i = 0; i < outnames.size(); ++i) {
```

Comment:
`outnames.size()` -> `ncols` for brevity

Reply:
Thanks, yes, all fixed.